x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -476,9 +478,21 @@ export class ExplorerUI extends React.Component {
                   </EuiFlexItem>
                   <EuiFlexItem grow={false} style={{ width: '170px' }}>
                     <EuiFormRow
-                      label={i18n.translate('xpack.ml.explorer.intervalLabel', {
-                        defaultMessage: 'Interval',
-                      })}
+                      label={
+                        <EuiToolTip
+                          content={i18n.translate('xpack.ml.explorer.intervalTooltip', {
+                            defaultMessage:
+                              'Show all anomalies or only the highest severity anomaly for each hour or day.',
```

Comment:
Is it worth trying to explain what the `auto` value? (Shows by hour or day depending on the selected time span - uses `hour` if less than 2 days selected, otherwise uses `day`).

Reply:
I'm not sure that the detail of auto is needed. 

One observation ... it is a better experience if you summarise your anomalies, hence the default is either hr/day. By leading with "Show all" are we encouraging "Show all"? .. because we should be encouraging to view as summarised.

Reply:
Thanks for the feedback!  I've altered the text in 709de27b1864e012a0b4ccc12fee638bfeb333ea